### PR TITLE
[Bugfix-16240] Explain how replace isn't recursive

### PR DESCRIPTION
**Pull request by livecodealice**
Added an explanation (and example) of how the replace command will not perform a replacement on overlapping matches due to how it searches for the string it is searching for.

### CODE REVIEW

File: docs/notes/Bugfix-16240.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Explained the non-recursive behaviour of the replace command in its entry.
```
Comment (livecodepanos) [inline docs/notes/Bugfix-16240.md]: The filename of this note should be `docs/notes/bugfix-16240.md`

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 43ec70e

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 43ec70ef6ce3e4f3f85415cf443f020ed96a2309

**Comment by livecode-vulcan** [comment]
😎 test success 43ec70ef6ce3e4f3f85415cf443f020ed96a2309

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/728)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/137)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/137)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/137)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/137)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2254)
* try-community-universal-ios-iphoneos13.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.2/builds/29)
* try-community-universal-ios-iphonesimulator13.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.2/builds/29)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2283)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2270)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2269)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2303)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2243)

**Pull request merged**